release.config.js

```diff
@@ -17,7 +17,22 @@ module.exports = {
         // }
     ],
     plugins: [
-        "@octorelease/changelog",
+        ["@octorelease/changelog", {
+            "displayNames": {
+                "cli": "Zowe CLI",
+                "core": "Core SDK",
+                "zosconsole": "z/OS Console SDK",
+                "zosfiles": "z/OS Files SDK",
+                "zosjobs": "z/OS Jobs SDK",
+                "zoslogs": "z/OS Logs SDK",
+                "provisioning": "Provisioning SDK",
+                "zostso": "z/OS TSO SDK",
+                "zosuss": "z/OS USS SDK",
+                "workflows": "Workflows SDK",
+                "zosmf": "z/OSMF SDK",
+                "cli-test-utils": "CLI Test Utils"
```

Comment:
Since the `secrets` package is published separately, I'm guessing it can't be added here?

Reply:
Good catch, I don't think it will hurt to add it now so that we don't forget to later 🙂 It just won't show up in our release notes until we stop publishing it separately and remove the `private: true` from its package.json.